scripts/check-format.sh

```diff
@@ -46,17 +46,40 @@ check_todo() {
   return 0
 }
 
+# Check that any calls that might panic have a comment noting why they're safe
+check_panic() {
+  local path="$1"
+  if [[ $path =~ "test" || $path =~ "examples/" || $path =~ "rinkey/" ]]; then
+    return 0
+  fi
+  for needle in "panic!(" "unwrap(" "expect(" "unwrap_err(" "expect_err(" "unwrap_none(" "expect_none("; do
+    local result
+    result=$(grep --with-filename --line-number "$needle" "$path" | grep --invert-match --regexp='safe:')
+    if [[ -n $result ]]; then
+      echo "$result"
+      return 1
+    fi
+  done
+  return 0
+}
+
 errcount=0
 for f in "${CODE_FILES[@]}"; do
   check_license "$f"
   errcount=$((errcount + $?))
-  # check_todo "$f"
+  check_todo "$f"
+  # TODO: check TODOs
   # errcount=$((errcount + $?))
```

Comment:
Should this still be commented out if the previous one isn't?

Reply:
I was aiming to have the message printed out (to make it easy to hunt remaining TODOs) without failing the CI builds -- I'll improve the comment.